cg/apps/orderform/excel_orderform_parser.py

```diff
@@ -195,12 +195,14 @@ def get_data_delivery(self, project_type: OrderType) -> str:
 
         if data_delivery == "fastq-qc":
             return DataDelivery.FASTQ_QC
+        if data_delivery == "fastq-+-analysis":
+            return DataDelivery.FASTQ_ANALYSIS
+        if data_delivery == "analysis-+-scout":
+            return DataDelivery.ANALYSIS_SCOUT
+        if data_delivery == "fastq-+-analysis-+-scout":
+            return DataDelivery.FASTQ_ANALYSIS_SCOUT
         if data_delivery == "fastq-qc-+-analysis":
             return DataDelivery.FASTQ_QC_ANALYSIS
```

Comment:
Instead of this if tree. Could you not make this a look up instead? Something like:
```
delivery_type_map = {"fastq-qc":  DataDelivery.FASTQ_QC,
"fastq-+-analysis": DataDelivery.FASTQ_ANALYSIS
etc etc}
return delivery_type_map[data_delivery]
```
It is usually faster and easier to read, test, extend and describe(type hint) than an if tree

Reply:
Good call 👍